.github/workflows/snapshot.yaml

```
@@ -53,3 +53,42 @@ jobs:
+         AGENTUITY_USER_ID: ${{ vars.AGENTUITY_USER_ID }}
+      steps:
+         - uses: actions/checkout@v4
+
+         - name: Setup Bun
+           uses: oven-sh/setup-bun@v2
+
+         - name: Install Agentuity CLI
+           run: curl -fsSL https://agentuity.sh | sh
+
+         - name: Generate CLI Auth Token
+           run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:1:15: Double quote to prevent globbing and word splitting

(shellcheck)